### ISSUE TITLE
Where is the batch normalization trick?

### ISSUE THREAD

**Issue by mmalekzadeh** [ssamot/DANN-keras#3]
Hey, thanks for the simple and sound code. Very useful!

I was wondering which part is implementing the idea of this paper:
"Revisiting batch normalization for practical domain adaptation"
